src/server/game/Handlers/MailHandler.cpp

```diff
@@ -22,8 +22,11 @@ bool WorldSession::CanOpenMailBox(uint64 guid)
 {
     if (guid == _player->GetGUID())
     {
-        sLog->outError("%s attempt open mailbox in cheating way.", _player->GetName().c_str());
-        return false;
+        if (_player->GetSession()->GetSecurity() < SEC_MODERATOR)
+        {
+            sLog->outError("%s attempt open mailbox in cheating way.", _player->GetName().c_str());
+            return false;
+        }
     }
```

Comment:
You can avoid the second if merging it with the first, like in this way:
```C++
    if (guid == _player->GetGUID() && _player->GetSession()->GetSecurity() < SEC_MODERATOR) {
        sLog->outError("%s attempt open mailbox in cheating way.", _player->GetName().c_str());
        return false;
    }
```

Reply:
also the `return` makes the `else` not needed

Reply:
so..:
```cpp
    if (guid == _player->GetGUID() && _player->GetSession()->GetSecurity() < SEC_MODERATOR) {
        sLog->outError("%s attempt open mailbox in cheating way.", _player->GetName().c_str());
        return false;
    }
    
    if (IS_GAMEOBJECT_GUID(guid)
```
?

Reply:
yes but { on next line after if :D

Reply:
Okay, I will see if I can do this tomorrow :)

Reply:
I don't want to do this, I tested it locally and this change breaks not only the command but the entire mailbox system